data/apply-updates-non-planet-tables.sql

```diff
@@ -83,9 +102,28 @@ UPDATE ne_10m_admin_1_states_provinces_lines
   SET mz_boundary_min_zoom = mz_calculate_min_zoom_boundaries(ne_10m_admin_1_states_provinces_lines.*)
   WHERE mz_calculate_min_zoom_boundaries(ne_10m_admin_1_states_provinces_lines.*) IS NOT NULL;
 
+UPDATE ne_10m_populated_places
+  SET mz_places_min_zoom = mz_calculate_min_zoom_places(ne_10m_populated_places.*)
+  WHERE mz_calculate_min_zoom_places(ne_10m_populated_places.*) IS NOT NULL;
 
-DO $$
-BEGIN
+UPDATE water_polygons SET mz_label_placement = ST_PointOnSurface(the_geom);
+UPDATE land_polygons SET mz_label_placement = ST_PointOnSurface(the_geom);
+UPDATE ne_110m_ocean SET mz_label_placement = ST_PointOnSurface(the_geom);
+UPDATE ne_50m_ocean SET mz_label_placement = ST_PointOnSurface(the_geom);
+UPDATE ne_10m_ocean SET mz_label_placement = ST_PointOnSurface(the_geom);
+UPDATE ne_110m_coastline SET mz_label_placement = ST_PointOnSurface(the_geom);
+UPDATE ne_50m_coastline SET mz_label_placement = ST_PointOnSurface(the_geom);
+UPDATE ne_10m_coastline SET mz_label_placement = ST_PointOnSurface(the_geom);
```

Comment:
Rebased the pr onto master, and updated to remove `mz_label_placement` from ne coastline tables in 5211aa2.